site/source/user-manual/connecting/connecting-lan/lan-os/lan-linux.rst

```diff
@@ -4,4 +4,20 @@
 Trusting Embassy CA on Linux
 ============================
 
-Nothing specific needs to be configured for the Linux environment.
+Nothing specific needs to be configured for the Linux environment at an OS level if you are only connecting over LAN through a **browser**.
+
+However, if you want to connect over LAN using a **native app** (e.g. the Bitwarden app or ``git``) you will need to trust the Root CA.
+
+Trusting Embassy CA on Ubuntu / Debian
+---------------
```

Comment:
These underlines `-----------------` must match the length of the text of the headline... an odd RST syntax thing

Reply:
@kn0wmad I fixed that up :+1: